README.md

```diff
@@ -322,3 +322,62 @@ The following statuses can be shown:
 - `No key` Signer name/key combination not in keys.txt, or key not known to GPG (which one of these it is, or both, will be listed under "Missing keys").
 - `Bad` Known key but invalid PGP signature.
 - `Mismatch`Correct PGP signature but mismatching binaries.
+
+ghwatch
+-------
+
+This is a script to watch your github notifications in the terminal. It will show a table that is auto-updated every 10 minutes (configurable). It can be exited by pressing <kbd>ESC</kbd> or <kbd>Ctrl-C</kbd>.
+
+### Dependencies
+
+The `github` python module is a required dependency for github API access. This can be installed for your user using `pip3 install --user github`, or globally using your distribution's package manager e.g. `apt-get install python3-github`.
```

Comment:
```suggestion
The `PyGithub` python module is a required dependency for github API access. This can be installed for your user using `pip3 install --user PyGithub`, or globally using your distribution's package manager e.g. `apt-get install python3-github`.
```

I am pretty sure this is called `PyGithub`?!

Reply:
Thanks, it's been a while since I installed it but fairly sure you're right.

Reply:
I can confirm that it is `PyGithub`.